pkgs/development/tools/pigeon/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+buildGoPackage {
+  pname = "pigeon";
+  version = "20190810-f3db42a662";
+
+  goPackagePath = "github.com/mna/pigeon";
+  goDeps = ./deps.nix;
+
+  src = fetchFromGitHub {
+    owner = "mna";
+    repo = "pigeon";
+    rev = "f3db42a662eded7550fc7cd11605d05311dfa30f";
+    sha256 = "1n0zqidwbqqfslrirpbqw14ylgiry6ggcp9ll4h8rf1chqwk6dhv";
+  };
+
+  meta = {
```

Comment:
would this be neater with `meta = with lib; {`

Reply:
I am trying to limit the scope of `with` as much as possible since it creates implicit dependencies. Also it would save 8 characters below and adds 10 characters.

Reply:
Fair enough, I was just going off of what is usually done